spec/spec_helper.rb

```diff
@@ -61,6 +61,7 @@ def puts(payload)
 
       LogStash::SETTINGS.set("queue.type", "memory")
       LogStash::SETTINGS.set("path.data", temp_directory)
+      LogStash::Config.module_eval { java_import org.logstash.config.ir.PipelineConfig }
```

Comment:
Note for reviewers:
this is a trick to assign the constant `LogStash::Config::PipelineConfig` to the new class else the gem `logstash-devutils` would break because it instantiate directly the class `LogStash::Config::PipelineConfig`

Reply:
Done